requirements.txt

```diff
@@ -4,9 +4,9 @@ django-nose==1.3
 logilab-common==0.63.2
 mock==1.0.1
 nose==1.3.4
-pylint==1.4.0
+pylint==1.4.1
 redis==2.10.3
-salt==0.17.1
+salt==2015.8.7
```

Comment:
Confirmed that the salt requirements was present only for the tests - bottle salinity was working even after removing this requirement. However, there are tests and for those we may need this dependency - in any case 2015.8.7 is the latest - so no further changes required.